mmv1/third_party/terraform/services/spanner/resource_spanner_database_test.go.tmpl

```diff
@@ -564,68 +572,33 @@ resource "google_spanner_database" "database" {
   ]
 
   encryption_config {
-  	kms_key_name = google_kms_crypto_key.example-key.id
+    kms_key_name = "%{key_name}"
   }
 
   deletion_protection = false
-
-  depends_on = [google_kms_crypto_key_iam_member.crypto-key-binding]
-}
-
-resource "google_kms_key_ring" "keyring" {
```

Comment:
Since keyrings are immutable, this would make one for each run. Then, in recording mode, even if the instance had a consistent name, the replay wouldn't make the request to create it (since it existed already).